### PR TITLE
Detect language API impl changes

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

**Review by mssfang — APPROVED** [review]

File: sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java [hunk 1]
```diff
@@ -35,11 +39,11 @@ DocumentResult setTextDocumentStatistics(TextDocumentStatistics textDocumentStat
         return this;
     }
 
-    public DocumentError getError() {
+    public Object getError() {
```
Comment (samvaity) [inline sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java:42]: need to finalize on the error object

Comment (mssfang) [reply sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java:42]: let's change to DocumentError

Comment (samvaity) [reply sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java:42]: do you still want this to be changed to DocumentError?

Comment (mssfang) [reply sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java:42]: I think it should be DocumentError. The output of the error content is DocumentError.

Comment (samvaity) [reply sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/DocumentResult.java:42]: Will resolve this in https://github.com/Azure/azure-sdk-for-java/issues/6559

**Pull request merged**